### ISSUE TITLE
Accessibility bug: email <input> missing id attribute

### ISSUE THREAD

**Issue by WestonThayer** [firebase/firebaseui-web#639]
### [REQUIRED] Describe your environment

  * Operating System version: macOS 10.14.6
  * Browser version: Chrome 77
  * Firebase UI version: 4.2.0
  * Firebase SDK version: 7.2.0

#### Steps to reproduce:

1. Go to the demo https://fir-ui-demo-84a6c.firebaseapp.com
2. Click **Sign in with email**
3. Inspect the email `<input>`

ACTUAL: There is a `<label>` next to it, with `for="email"`, but the input does not have `id="email"`. This means screen readers or other assistive technologies do not associate the label with the input

EXPECTED: It should be accessible

If you want to experience the issue with a screen reader, you can do it on macOS by starting VoiceOver, and TABing to the field. VoiceOver announces it as edible text, and gives no indication that this field is for you to input your email address.

![image](https://user-images.githubusercontent.com/709153/66875626-d8d2b880-ef63-11e9-93a9-dfd4422bbe3d.png)

**Comment by sophieheb** [comment]
Have created a suggested fix for this here: https://github.com/firebase/firebaseui-web/pull/655

**Comment by dackers86** [comment]
Closing as this issue has been resolved.
